apps/data-pipeline/map-scraper/src/index.ts

```diff
@@ -0,0 +1,140 @@
+import { exit } from "node:process";
+import { database } from "@packages/db";
+import { mapLocation } from "@packages/db/schema";
+
+interface LocationListData {
+  catId: number;
+  lat: number;
+  lng: number;
+  id: number;
+  name: string;
+}
+
+interface LocationDetailData {
+  mediaUrlTypes: string[];
+  mediaUrls: string[];
+}
+
+interface BuildingLocation {
+  name: string;
+  lat: number;
+  lng: number;
+  imageURLs: string[];
+}
+
+const CATEGORIES: Set<number> = new Set([
```

Comment:
can you leave a note about how we decided that hard-coding these is mandatory?

Reply:
unresolving - you need to explicitly make it clear that you had to hardcode the api key AND non-secret constants like the categories and UCI map ID